<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>165</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>165</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>165</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gardener</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layersFrame">
     <item>
      <layout class="QVBoxLayout" name="imageVerticalLayout">
       <item>
        <widget class="QLabel" name="imageLabel">
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsMapLayerComboBox" name="imageLayerComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="indexVerticalLayout">
       <item>
        <widget class="QLabel" name="indexLabel">
         <property name="text">
          <string>Index</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsMapLayerComboBox" name="indexLayerComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsFrame">
     <item>
      <widget class="QPushButton" name="suppressButton">
       <property name="text">
        <string>Suppress</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paramsButton">
       <property name="text">
        <string>Parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statLabel">
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statHorizontalLayout" stretch="3,1">
     <item>
      <widget class="QgsRasterBandComboBox" name="imageRasterBandComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="showButton">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsRasterBandComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsrasterbandcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
